containerbuddy/memcached.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
There's an argument for prefixing the scripts with the event they match, like `onstart` and `onchange`. `onstart.sh` already matches ;-)